### ISSUE TITLE
DL solution without any decisions regarding the neural architecture of the model

### ISSUE THREAD

**Issue by kvartet** [microsoft/nni#3830]
**Describe the issue**:

does NNI have a solution for DL that allows me to feed in the inputs, outputs and evaluate the model while leaving me out of any decisions regarding the neural architecture of the model? Most of the examples I have seen force me to make some layer choices

-------------------------------
This issue was asked on Gitter, author: @8W9aG

**Comment by ultmaster** [comment]
Good point. We will design and try to support this feature in future releases.

**Comment by 8W9aG** [comment]
Thanks for looking into this, some features that would be great are:

* Feed in a current model that works, and maybe start the search from
* Search is never-ending (ie it keeps going further and further from the current model)

Is there anything I can do to help this along?
